dist/common/scripts/scylla_util.py

```diff
@@ -457,6 +457,23 @@ def is_unused_disk(dev):
     except OSError:
         return False
 
+PART_UUID_BIOS_BOOT_PARTITION = '21686148-6449-6e6f-744e-656564454649'
+PART_UUID_EFI_SYSTEM_PARTITION = 'c12a7328-f81f-11d2-ba4b-00a0c93ec93b'
+
+def is_partition(dev):
+    return os.path.exists('/sys/class/block/{dev}/partition'.format(dev=dev.replace('/dev/', '')))
+
+def get_partition_uuid(dev):
+    return out(f'lsblk -n -oPARTTYPE {dev}')
+
+def is_system_partition(dev):
+    if not is_partition(dev):
+        return False
+    uuid = get_partition_uuid(dev)
+    if uuid == PART_UUID_BIOS_BOOT_PARTITION or uuid == PART_UUID_EFI_SYSTEM_PARTITION:
+        return True
+    return False
```

Comment:
You could introduce an `SYSTEM_PARTITION_UUIDS` list and then just make this:

```python
return uuid in SYSTEM_PARTITION_UUIDS
```

Reply:
@syuu1228 ^